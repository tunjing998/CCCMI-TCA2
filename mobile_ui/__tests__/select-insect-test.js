import React from 'react';
import renderer from 'react-test-renderer';

import {NativeModules as RNNativeModules} from 'react-native';
RNNativeModules.UIManager = RNNativeModules.UIManager || {};
RNNativeModules.UIManager.RCTView = RNNativeModules.UIManager.RCTView || {};
RNNativeModules.RNGestureHandlerModule = RNNativeModules.RNGestureHandlerModule || {
  State: {BEGAN: 'BEGAN', FAILED: 'FAILED', ACTIVE: 'ACTIVE', END: 'END'},
  attachGestureHandler: jest.fn(),
  createGestureHandler: jest.fn(),
  dropGestureHandler: jest.fn(),
  updateGestureHandler: jest.fn(),
};
RNNativeModules.PlatformConstants = RNNativeModules.PlatformConstants || {
  forceTouchAvailable: false,
};

test('selectInsect1 snapShot', () => {
  const snap = renderer.create(<selectInsect1 />).toJSON();
  expect(snap).toMatchSnapshot();
});

test('selectInsect2 snapShot', () => {
  const snap = renderer.create(<selectInsect2 />).toJSON();
  expect(snap).toMatchSnapshot();
});
