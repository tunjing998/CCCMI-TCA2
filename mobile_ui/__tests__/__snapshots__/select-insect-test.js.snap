// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`selectInsect1 snapShot 1`] = `<selectInsect1 />`;

exports[`selectInsect2 snapShot 1`] = `<selectInsect2 />`;
